burn-import/tests/data/model1/model1.rs

```diff
@@ -1,67 +1,59 @@
-// Generated from tests/data/model1/model1.onnx by burn-import
-
-
-use burn::tensor::activation::log_softmax;
-use burn::tensor::activation::relu;
+// Generated by integration tests
+use burn::nn::conv::Conv2d;
+use burn::nn::conv::Conv2dConfig;
+use burn::nn::BatchNorm;
+use burn::nn::BatchNormConfig;
+use burn::nn::Linear;
+use burn::nn::LinearConfig;
 use burn::{
     module::Module,
-    nn,
-    record::{DefaultRecorder, Recorder},
     tensor::{backend::Backend, Tensor},
 };
 
-pub const INPUT1_SHAPE: [usize; 4usize] = [1usize, 1usize, 8usize, 8usize];
-pub const OUTPUT1_SHAPE: [usize; 2usize] = [1usize, 288usize];
```

Comment:
Do we still want to provide some facility to know the input shape? The user may have to guess or know in advance but it's part of the model.

Reply:
The input and output shapes may vary, if you want to use a batch size of 1 or 32 it shouldn't matter, so I prefer not exporting it, making its usage similar to other burn modules. The best solution would be to export comments explaining each dimension (like batch_size, seq_length, channel, width, etc.) but I'm not sure this information is present in ONNX.

Reply:
We could just document it in the source somewhere. I am not sure really. Or maybe we could provide some tooling that summarizes the model like this: http://www.xavierdupre.fr/app/onnxruntime/helpsphinx/auto_examples/plot_load_and_predict.html

We could think of a solution and do it later. I am okay with this.